### PR TITLE
Cockpit 198

### PR DESCRIPTION
**Pull request by martinpitt**
Rendered preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-198.html

### CODE REVIEW

File: _posts/2019-07-10-cockpit-198.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+slug: cockpit-198
+category: release
+summary: Cockpit with PatternFly 4 user interface design update
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 198.
+
+### New user interface design
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: I'd mention pf4 in the heading

File: _posts/2019-07-10-cockpit-198.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+![system page](/images/system-pf4.png)
+
+### SELinux: Show changes
+
+The SELinux page now has a new section "System Modifications" which shows all
+policy settings that were made to the system (with Cockpit or otherwise):
+
+![SELinux modifications](/images/selinux-modifications.png)
+
+The "View Automation Script" link will show a dialog with a Shell script that
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: random capital "Shell"?

File: _posts/2019-07-10-cockpit-198.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+can be used to apply the same changes to other machines:
+
+![SELinux automation script](/images/selinux-autoscript.png)
+
+### Machines: Deletion of Virtual Networks
+
+On the Machines page's "Virtual Networks" page, networks can now be deleted:
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: > Machines page's "Virtual Networks" page

oy

Maybe something like "The Virtual Networks section of the Machines page now supports deleting networks"?

File: _posts/2019-07-10-cockpit-198.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+![delete virtual network](/images/machines-delete-network.png)
+
+### Machines: Support more disk types
+
+When creating a new VM, the disk can now be on a storage pool type other than a
+plain file. This now supports iSCSI, LVM, and physical volumes.
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: "This now supports" sounds awkward.  Maybe "The newly supported types are..." or something?

File: _posts/2019-07-10-cockpit-198.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+[cockpit-podman](https://github.com/cockpit-project/cockpit-podman), and points
+out that this page really is about Docker only, not any other container
+technology.
+
+### Web server: More flexible https redirection for proxies
+
+[cockpit-ws](https://cockpit-project.org/guide/latest/cockpit-ws.8.html) now
+supports redirecting unencrypted http to https (TLS) even when running in
+`--no-tls` mode. Use this when running cockpit-ws behind a reverse http proxy
+that also supports https, but does no redirection from http to https by itself.
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: "but doesn't handle the redirection ... for itself"

**Comment by martinpitt** [comment]
Thanks @allisonkarlitskaya for the review! Updated to address all points.

**Comment by garrett** [comment]
It's a bit misleading to say Cockpit is using PatternFly 4, as it doesn't really.

We've restyled all of Cockpit to match the look of PatternFly 4 and will be porting parts of Cockpit to PatternFly 4 as time goes on. Restyling Cockpit allows us to move gradually instead of having a mishmash of difrently-styled widgets or trying to adapt everything at once.

**Comment by martinpitt** [comment]
@garrett: Right, I was carefully avoiding to say "uses PatternFly 4", just that it uses its design. Do you have an idea how to make that less ambiguous?

**Comment by garrett** [comment]
> Do you have an idea how to make that less ambiguous?

Well, I guess we could use my paragraph, or something like it:

> We've restyled all of Cockpit to match the look of PatternFly 4 and will be porting parts of Cockpit to PatternFly 4 as time goes on. Restyling Cockpit allows us to move gradually instead of having a mishmash of differently-styled widgets or trying to adapt everything at once.

**Comment by martinpitt** [comment]
Thanks @garrett! I updated the paragraph again, how does it look now?

File: _posts/2019-07-10-cockpit-198.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+summary: Cockpit with PatternFly 4 user interface design update
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 198.
+
+### PatternFly4 user interface design
+
+Cockpit got restyled to match the
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: "got" is really very informal.  maybe "has been".

Comment (garrett) [inline _posts/2019-07-10-cockpit-198.md]:
```suggestion
Cockpit has been restyled to match the
```

Using "got" here like this is what a lot of people would say when speaking, but I don't think it's really proper to use when writing. I'm adding a quick change, although I'm not a fan of "has been".

@allisonkarlitskaya: Do you have a better suggestion?

File: _posts/2019-07-10-cockpit-198.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+[cockpit-podman](https://github.com/cockpit-project/cockpit-podman), and points
+out that this page really is about Docker only, not any other container
+technology.
+
+### Web server: More flexible https redirection for proxies
+
+[cockpit-ws](https://cockpit-project.org/guide/latest/cockpit-ws.8.html) now
+supports redirecting unencrypted http to https (TLS) even when running in
+`--no-tls` mode. Use this when running cockpit-ws behind a reverse http proxy
+that also supports https, but does no redirection from http to https for itself.
```
Comment (allisonkarlitskaya) [inline _posts/2019-07-10-cockpit-198.md]: You marked this resolved, but didn't change anything as far as I can tell?

File: _posts/2019-07-10-cockpit-198.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+release regularly.  Here are the release notes from version 198.
+
+### PatternFly4 user interface design
+
+Cockpit got restyled to match the
+[PatternFly 4](https://pf4.patternfly.org/components/Button/examples/)
+User Interface design, including the
+[Red Hat Text and Display fonts](https://github.com/RedHatOfficial/RedHatFont).
+
+This aligns Cockpit with other web user interfaces that use
```
Comment (garrett) [inline _posts/2019-07-10-cockpit-198.md]:
```suggestion
This style refresh aligns Cockpit with other web user interfaces that use
```

It's almost always a good idea to say what "this" is, especially when it's in a different paragraph.

File: _posts/2019-07-10-cockpit-198.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+[PatternFly](https://www.patternfly.org/), such as
+[OpenShift 4](https://www.redhat.com/en/openshift-4).
+
+Over time, Cockpit will be ported to actually use PatternFly 4 widgets, but
+this restyle allows us to do this gradually.
```
Comment (garrett) [inline _posts/2019-07-10-cockpit-198.md]:
```suggestion
this restyle allows us to change Cockpit gradually.
```

File: _posts/2019-07-10-cockpit-198.md (hunk 7 above)
Comment (martinpitt) [reply _posts/2019-07-10-cockpit-198.md]: I replaced "by itself" with "for itself", that's how I understood your remark?

**Pull request merged**